ww4ff/align/stub.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from .base import AlignedTranscription
+
+
+__all__ = ['StubAligner']
+
+
+class StubAligner:
```

Comment:
just out of curiosity, where does the name StubAligner come from?

Reply:
It's a class stub that just produces some mock alignment.